.cspell.yaml

```
@@ -0,0 +1,54 @@
+# For settings, see
+# https://www.streetsidesoftware.com/vscode-spell-checker/docs/configuration/
+version: '0.2'
+language: en-US
+caseSensitive: false
+allowCompoundWords: true
+useGitignore: true
+ignorePaths:
+  - '**/*.svg'
+files:
+  - '**/*.md'
+patterns:
+  - name: CodeBlock
+    pattern: |
+      /
+          ^(\s*[~`]{3,})   # code-block start
```

are `~` code blocks a thing?

```suggestion
          ^(\s*[`]{3,})   # code-block start
```

---

Based on the docs they are https://www.markdownguide.org/extended-syntax/#fenced-code-blocks & it is also the regex from otel.io.